commit d882a1b68d4692b740149516d8a7ed226a669574
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-05-22T18:52:02+00:00 (5 months ago)
    
    TRITON-1690 update cloudapi node-mahi dep for new RBAC features
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/auth.js b/lib/auth.js
index 2826fd86..9d95ae2b 100644
--- a/lib/auth.js
+++ b/lib/auth.js
@@ -627,7 +627,8 @@ function authorize(req, res, next) {
                 'user-agent': req.headers['user-agent'],
                 sourceip: req.connection._xff,
                 method:   req.method.toLowerCase(),
-                activeRoles: activeRoles.map(function (r) { return (r.uuid); })
+                activeRoles: activeRoles.map(function (r) { return (r.uuid); }),
+                datacenter: req.config.datacenter_name
             }
         };
 
diff --git a/package.json b/package.json
index fad2cdba..67002506 100644
--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
         "keyapi": "git+https://github.com/joyent/keyapi.git#e14b3d582e1d9d338b7082d61f34ba8d1bbc540a",
         "krill": "1.0.1",
         "libuuid": "0.2.1",
-        "mahi": "2.0.1",
+        "mahi": "2.3.0",
         "mime": "1.2.7",
         "nodemailer": "0.7.1",
         "nopt": "2.0.0",
